### PR TITLE
feat(rm): _add_missing_reads

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### Added

-

### Changed

- Remove _add_missing_reads

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Review by seallard — APPROVED** [review]
Nice!

File: cg/meta/orders/case_submitter.py [hunk 1]
```diff
@@ -96,23 +96,15 @@ def _validate_case_names_are_unique(
     def submit_order(self, order: OrderIn) -> dict:
         """Submit a batch of samples for sequencing and analysis."""
         result = self._process_case_samples(order=order)
-        for case_obj in result["records"]:
```
Comment (seallard) [inline cg/meta/orders/case_submitter.py:99]: Can we remove this?

Comment (henrikstranneheim) [reply cg/meta/orders/case_submitter.py:99]: Sure! @islean @Vince-janv Any opinions?

Comment (islean) [reply cg/meta/orders/case_submitter.py:99]: If it is only there for the LOG.info I am all for removing it!

**Comment by henrikstranneheim** [comment]
@Clinical-Genomics/sysdev Anyone that could help me test this one? We are only removing code , but maybe submitting an order should suffice.

**Comment by ChrOertlin** [comment]
@henrikstranneheim I will have a look at this. There is likely a refactoring of the submitters coming https://github.com/Clinical-Genomics/cg/pull/3574 . I will try to test and merge before that. Nice if we can remove the function before than.

**Comment by ChrOertlin** [comment]
# Tests
Using balsamic.json in tests/fixtures/cg_weborders/

## added to lims:
image

## ticket created
image

## in status
image

**Comment by ChrOertlin** [comment]
@Vince-janv Can you double check this for me here?

**Review by Vince-janv — APPROVED** [review]
Nice! @Karl-Svard confirmed that the function is not working and that the value is set later by LIMS

**Comment by henrikstranneheim** [comment]
Thx for the help! Then I will merge this early next week  if you do not beat me to it.

**Comment by ChrOertlin** [comment]
> Thx for the help! Then I will merge this early next week if you do not beat me to it.

@henrikstranneheim I think we can close this pr. I made some headway over the weekend and get the refactoring tested here: https://github.com/Clinical-Genomics/cg/pull/3574

I made sure to remove the _add_missing_reads function there. If you agree you can close this PR, in favour of addressing it in the other.

**Comment by henrikstranneheim** [comment]
Great!

**Pull request closed without merging**